src/App.js

```diff
@@ -1,29 +1,8 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './component/calculator';
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  return (<Calculator />);
```

Comment:
I would recommend wrapping `Calculator` component inside a `div` tag, so you can apply some styling to match the given design exactly even though you design is matching based on colors. Right now, your calculator design is spreading through out the width of any screen size. 

| Your design  | Provided design |
| ------------- | ------------- |
| ![image](https://github.com/shihabrafio/Math-Magician/assets/33268927/9d1a5267-1015-4bb9-81dd-b57365c0cd27) | ![image](https://github.com/shihabrafio/Math-Magician/assets/33268927/2a7f54dc-91c0-4698-b891-2c4500ea7d0d) |